dotCMS/src/main/java/com/dotcms/rest/api/v1/user/UserResource.java

```diff
@@ -403,7 +403,7 @@ private void updateLoginAsSessionInfo(final HttpServletRequest request, final Ho
 		final String userToImpersonate = (UtilMethods.isSet(loginAsUserId) ? loginAsUserId : principalUserId);
```

Comment:
should this change have a postman test or at least a unit/integration test?

Reply:
done